code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2915,6 +2915,79 @@
 	if(T.seed && !T.dead)
 		T.health += 50
 
+//Just for fun
+var/procizine_call = ""
+var/list/procizine_args = list()
+
+/client/proc/set_procizine_call()
+	set name = "Set procizine call"
+	set category = "Fun"
+	if(!check_rights(R_DEBUG))
+		return
+
+	procizine_call = input("Proc path to call on target reaction, eg: /proc/fake_blood (To make effective, add the reagent procizine to the atom)","Path:", null) as text|null
+	if(!procizine_call)
+		return
+
+	var/argnum = input("Number of arguments","Number:",0) as num|null
+	if(!argnum && (argnum!=0))
+		return
+
+	procizine_args.Cut()
+	procizine_args.len = argnum // Expand to right length
+
+	var/i
+	for(i = 1, i < argnum + 1, i++) // Lists indexed from 1 forwards in byond
+		procizine_args[i] = variable_set(src)
+
+/datum/reagent/procizine
+	name = "Procizine"
+	id = PROCIZINE
+	description = "It is a mystery!"
+	reagent_state = REAGENT_STATE_LIQUID
+	color = "#C8A5DC" //rgb: 200, 165, 220
+	density = ARBITRARILY_LARGE_NUMBER
+	specheatcap = ARBITRARILY_LARGE_NUMBER
+	var/procname = ""
+	var/list/procargs = list()
+
+/datum/reagent/procizine/New()
+	..()
+	procname = procizine_call
+	procargs = procizine_args.Copy()
+
+/datum/reagent/procizine/proc/call_proc(var/atom/A)
+	if(procname && hascall(A, procname))
+		call(A,procname)(arglist(procargs))
+
```

Comment:
This should probably have give a warning message to admins and remove the reagent if something slips through hascall().

Reply:
What do you mean?

Reply:
No really what do you mean